template/config/storybook/components/playground.vue

```diff
@@ -0,0 +1,153 @@
+<script lang="ts">
+import { PropOptions, VueConstructor } from 'vue';
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import { StoryProp } from '.storybook/modules';
+// import { StoryProp } from '@/story';
```

Comment:
@taslangraham  should this be commented out?

Reply:
@OrandiH it should have actually been removed completely.